openapi/endpoints/cards/create-card-async.yaml

```diff
@@ -0,0 +1,39 @@
+post:
+  tags:
+    - card
+  summary: Create a card
+  description: |
+    Create a card under a specified card program and affiliated with a funding source.
```

Comment:
"affiliated with a funding source" sounds weird to me, what about
"linked to a funding source"?